deviceComunicationDocumentation.MD

```diff
@@ -836,8 +770,7 @@ Nepoužitý.
 
 #### 4.byte a 5.byte - typ dat
```

Comment:
To odesílání mi přijde trochu matoucí, nepopsat to ve stejném stylu jako uplink? Tedy že  existuje hlavička a typ zprávy a ne že typ dat který se liší podle nějakého byte. Spíš že se liší podle typu zprávy aby to bylo jednotné s uplinkem.

Reply:
upraveno